containers/example-compose.yaml

```diff
@@ -37,3 +59,9 @@ services:
     volumes:
       - ./ui-config:/home/freetak/data:rw,Z
     network_mode: "host"
+    environment:
+      # Change the following 3 lines to the FTS external IP
```

Comment:
the more I think about this the less I like it, I don't think we can conclusively say this for all use cases if the network is even slightly atypical.